src/App.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import './App.css';
+import Modal from './Components/Modal';
+
+function App() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const appTitle = `Componente modal con React`
+  const buttonTxt = `Abrir`;
+  return (
+    <div className="App">
+      <h1 className='appHeader'>{appTitle}</h1>
+      <button className="openModal" onClick={() => {setIsModalOpen(true);}}>{buttonTxt}</button>
```

Comment:
```suggestion
      <button className="openModal" onClick={() => {
        setIsModalOpen(true);
       }}>
         {buttonTxt}
      </button>
```

or

```suggestion
      <button className="openModal" onClick={() => setIsModalOpen(true)}>{buttonTxt}</button>
```

No biggie. If you use a code formatter like [Prettier](https://prettier.io) won't be a problem.

Reply:
Thank you! I have installed Prettier and it is easy to use to format my code. I am using it on my last commit.
Yes I had read in some website that if the return only have one line you can skip the braces, that was a mistake...